pilot/pkg/bootstrap/server.go

```diff
@@ -994,20 +989,15 @@ func getDNSNames(args *PilotArgs, host string) []string {
 	if args.Revision != "" && args.Revision != "default" {
 		knownHosts = append(knownHosts, "istiod"+"-"+args.Revision)
 	}
-
+	knownSans := make([]string, 0, 2*len(knownHosts))
 	for _, altName := range knownHosts {
-		name := fmt.Sprintf("%v.%v.svc", altName, args.Namespace)
-		exist := false
-		for _, cHost := range cHosts {
-			if name == host || name == cHost {
-				exist = true
-			}
-		}
-		if !exist {
-			dnsNames = append(dnsNames, name)
-		}
+		knownSans = append(knownSans,
+			fmt.Sprintf("%s.%s", altName, args.Namespace),
```

Comment:
why are we adding this new one?

Reply:
It is valid though

Reply:
I cannot recall why it was not there

Reply:
It is but not sure we need all of them? In theory a user could have `com` namespace for example, now we give it `istiod.com` cert, etc. Seems safer to keep with what we have, especially since we do not have a reason

Reply:
ok